examples/next-appDir/README.md

```diff
@@ -0,0 +1,24 @@
+## Setting up
+To try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
+in the `.env.local`. You can just copy `.env` file located in this directory.
+
+More info about the API key you can find on the [documentation page](https://dev.fingerprint.com/docs/js-agent#agent-initialization).
+
+### Installing dependencies
+1. Go to the root of the project, install dependencies with `yarn install` and build it with `yarn build`.
+2. Come back to the example folder (`cd examples/next`) and run `yarn install`.
+
+After you've completed all the steps and inserted the API key, you can run:
+```shell
+yarn dev
+```
+
+Runs the app in the development mode.\
+Open [http://localhost:3002](http://localhost:3002) to view it in the browser.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
```

Comment:
This is a nice touch, but since the focus of the example is to demonstrate usage inside the `app` directory, I would consider linking to resources specifically focused on that, probably things in the Next 13 beta docs: https://beta.nextjs.org/docs/getting-started